File: server/java/test/hz/message/MessageCreatorTest.java
package hz.message;

import static org.junit.Assert.*;

import org.junit.Test;

/**
 * 测试MessageCreator
 */
public class MessageCreatorTest {
	/** mysql用户名 */
	private final String _user = "root";
	/** mysql用户密码 */
	private final String _password = "Ningning~1";
	/** mysql数据库名 */
	private final String _database = "test_message";
	
	/**
	 * 测试MessageCreator.createMessageByMysql()
	 * 
	 * @throws Exception 错误异常
	 */
	@Test
	public void test_createMessageByMysql() throws Exception {
		// 执行createMessageByMysql
		Message message = MessageCreator.createMessageByMysql(_user, _password, _database);

		// 测试创建对象不为空
		assertNotNull(message);
	}
}

File: server/node/test/hz/message/message_creator_test.js
#!/usr/bin/env node

'use strict';

// 引入模块
const assert = require('assert');
const message_creator = require('../../../src/hz/message/message_creator.js');

/** mysql用户名 */
const user = 'root';
/** mysql用户密码 */
const password = 'Ningning~1';
/** mysql数据库名 */
const database = 'test_message';

describe('message.Message', function() {
	it('message.Message.request()', function(done) {
		this.timeout(0);
		
		// 执行createMessageByMysql
		message_creator.createMessageByMysql(user, password, database, function(err, msg) {
			// 测试创建对象为message
            assert(!err);
			assert('request' in msg);
            
            done();
		});
	});
});
